webpack/assets/javascripts/react_app/components/AutoComplete/AutoCompleteActions.js

```diff
@@ -99,16 +135,17 @@ const requestSuccess = ({
   });
 };
 
-const requestFailure = ({ error, id, dispatch }) =>
+const requestFailure = ({ error, id, dispatch, isVisible = true }) => {
   dispatch({
     type: AUTO_COMPLETE_FAILURE,
     payload: {
       results: [],
-      error: error.message || error,
+      error: { message: error, isVisible },
```

Comment:
The `error` is an object so imo it shouldn't be called `message`.
I think `payload: { error, isErrorVisible: isVisible }` represents the values better.